pandas/core/internals/ops.py

```diff
@@ -54,6 +56,8 @@ def operate_blockwise(
 
     res_blks: list[Block] = []
     for lvals, rvals, locs, left_ea, right_ea, rblk in _iter_block_pairs(left, right):
+        lvals = ensure_wrapped_if_datetimelike(lvals)
+        rvals = ensure_wrapped_if_datetimelike(rvals)
```

Comment:
@jbrockmendel I suppose those can be removed now, since the Blocks always store DatetimeArray/TimedeltaArray ?

Reply:
i think so, yes